docs/crio.8.md

```diff
@@ -109,7 +109,7 @@ crio [GLOBAL OPTIONS] command [COMMAND OPTIONS] [ARGUMENTS...]
 
 **--additional-devices**="": Devices to add to the containers  (default: [])
 
-**--apparmor-profile**="": Name of the apparmor profile to be used as the runtime's default. The default profile name is 'crio-default-' followed by the version string of CRI-O. (default: "crio-default-1.17.0-dev"). This only takes effect if the user does not specify a profile via the Kubernetes Pod's metadata annotation.
+**--apparmor-profile**="": Name of the apparmor profile to be used as the runtime's default. The default profile name is 'crio-default-' followed by the version string of CRI-O. (default: "crio-default-1.18.0-dev"). This only takes effect if the user does not specify a profile via the Kubernetes Pod's metadata annotation.
```

Comment:
IDK how, but it would sure be nice to automatically generate the version here in these files.

Reply:
I would like to get rid of this AppArmor version from my personal preference, it's too error prone. I'll follow up in a dedicated PR.